influxql/engine.go

```diff
@@ -204,6 +204,50 @@ func (m *MapReduceJob) Execute(out chan *Row, filterEmptyResults bool) {
 	out <- row
 }
 
+// mergeOutputs merges two sorted slices of rawQueryMapOutput such that duplicate
```

Comment:
It's happens from removing the hard coding of `sg.shards[0]`here: https://github.com/influxdb/influxdb/blob/master/tx.go#L147

It's now looping over each shard.